scripts/miniapps/SNAP.md

```diff
@@ -0,0 +1,136 @@
+# Instruction for building and running SNAP
+
+Implementation: Fortran, MPI+OpenMPI, uses MPI_THREAD_SERIALIZE
+
+Useful links:
+- GitHub repository: https://github.com/losalamos/SNAP
+- Useful build and run info: http://www.nersc.gov/users/computational-systems/cori/nersc-8-procurement/trinity-nersc-8-rfp/nersc-8-trinity-benchmarks/snap
+
+1. Download from the GitHub repository
+   ```
+% git clone https://github.com/losalamos/SNAP.git
+   ```
+
+   README.md in the top level directory contains some useful information
+
```

Comment:
Does the README file contain useful information or not?
And what information might be useful in the README file?